client/modules/IDE/components/Editor/index.jsx

```diff
@@ -543,14 +541,20 @@ class Editor extends React.Component {
                     this.props.closeProjectOptions();
                   }}
                 >
-                  <LeftArrowIcon focusable="false" aria-hidden="true" />
+                  <IconButton
+                    onClick={this.props.expandSidebar}
```

Comment:
I think these lines (544-545) probably aren't needed here, since they're covered by the parent `button` element right above. If added, the `IconButton` implementation could probably look more something like: `<IconButton focusable="false" aria-hidden="true" />`